cli/dts/lib.deno.ns.d.ts

```diff
@@ -1893,19 +1893,35 @@ declare namespace Deno {
       signal?: number;
     };
 
-  export interface RunOptions {
+  export type RunOptions = 
+    | RunOptionsAttached
+    | RunOptionsDetached;
+
+  interface RunOptionsCommon {
     /** Arguments to pass. Note, the first element needs to be a path to the
      * binary */
     cmd: string[] | [URL, ...string[]];
     cwd?: string;
+    detached?: boolean;
     env?: {
       [key: string]: string;
     };
+  }
+
+  interface RunOptionsAttached extends RunOptionsCommon {
+    detached?: false,
     stdout?: "inherit" | "piped" | "null" | number;
     stderr?: "inherit" | "piped" | "null" | number;
     stdin?: "inherit" | "piped" | "null" | number;
   }
 
+  interface RunOptionsDetached extends RunOptionsCommon {
+    detached?: true,
+    stdout?: "null";
+    stderr?: "null";
+    stdin?: "null";
```

Comment:
Are you sure these options aren't still applicable? What's the runtime behaviour if you pass `"inherit"` vs `"null"` here?

Reply:
Actually inherit is still good as it's the default if it's undefined. I just forgot to add it as an option.

Running a detached process with inherit will still log the results to the console, even when the process spawner quits.

Reply:
Passing an `"rid"` also works. So I will disallow only `"piped"`